### PR TITLE
h_long_5

### PR DESCRIPTION
**Pull request by erinlincoln**
- Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/h_long_5.ncl
- Original NCL plots: https://www.ncl.ucar.edu/Applications/Images/h_long_5_lg.png 
- Issue #345 
image

I am going to play around with the latitude to see if I can find the right one!

### CODE REVIEW

**Comment by erinlincoln** [comment]
I think using isel lat 26 matches? No idea why it is that specifically but the data looks the same.

**Comment by erinlincoln** [comment]
Updated figure:
h_long_5_fig_1

**Comment by anissa111** [comment]
I think I found a method of getting the same result without manually selecting the lat

```
U = ds.U.isel(time=0)
U = U.sel(lat=-16, method="nearest")
```

produces:
image

**Comment by erinlincoln** [comment]
That makes sense! I will change that on my end.

**Review by Jiaqi-beep — COMMENTED** [review]
Looks really good to me! Super nitpicky, but it seems to me that the maintitle and the x/y labels are the same fontsize in the original plot.

File: Plots/Contours/NCL_h_long_5.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+# Plot contour lines
+lines = U.plot.contour(ax=ax,
+                       levels=levels,
+                       colors='black',
+                       linewidths=0.5,
+                       linestyles='solid',
+                       add_labels=False)
+
+# Label contour lavels at -10, 0, and 10 and set their backgrounds to be white
```
Comment (anissa111) [inline Plots/Contours/NCL_h_long_5.py:57]:
```suggestion
# Label contour levels at -10, 0, and 10 and set their backgrounds to be white
```

File: Plots/Contours/NCL_h_long_5.py [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+    xticks=np.linspace(-180, 178, 7),
+    xticklabels=['180', '120W', '60W', '0', '60E', "120E", ""],
+    ylim=ax.get_ylim()[::-1],
+    yticks=U["lev"])
+
+# Change formatter or else tick values will be in exponential form
+ax.yaxis.set_major_formatter(ScalarFormatter())
+
+# Use geocat.viz.util convenience function to add major tick lines with no
+# minor ticks on lefthand side y axis and some minor ticks on the x axis
```
Comment (anissa111) [inline Plots/Contours/NCL_h_long_5.py:93]:
```suggestion
# minor ticks on left hand side y axis and some minor ticks on the x axis
```

File: Plots/Contours/NCL_h_long_5.py [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+                             lefttitle=U.long_name,
+                             lefttitlefontsize=14,
+                             righttitle=U.units,
+                             righttitlefontsize=14,
+                             ylabel=U.lev.long_name,
+                             labelfontsize=16)
+
+# Create second y-axis to show geo-potential height.
+# Currently we're using arbitrary values for height as we haven't figured out
+# how to make this work properly yet
```
Comment (anissa111) [inline Plots/Contours/NCL_h_long_5.py:112]: I would consider making this either a block comment or formatting it to be a text block in the documentation build

**Review by anissa111 — COMMENTED** [review]
I agree with @Jiaqi-beep that you might want to bump down the title size and up both the y-axis label sizes. Maybe also scoot the height label a little farther to the right.

**Review by anissa111 — COMMENTED** [review]
I like the font size updates! There's one other thing though, and it's absolutely my bad. My last review said to make lines 110 - 113 a text block, but doing this has made the plot in the documentation generation appear above that line instead of at the end of the script. 

image

I forgot that this is a thing that sometimes happen if you break up the plotting section with anything, please feel free to revert back to an in code block comment!

**Comment by erinlincoln** [comment]
When I make it a multi-line comment block, it fails the pre-commit check. Should I leave it as multiple lines of comments?

**Comment by anissa111** [comment]
> When I make it a multi-line comment block, it fails the pre-commit check. Should I leave it as multiple lines of comments?

Yup, that works, sorry about making you change it haha

**Review by anissa111 — APPROVED** [review]
Looks great!

**Pull request merged**